src/components/clientDashboard/ProfileLayout.tsx

```diff
@@ -1,50 +1,33 @@
-import { Alert, AlertProps, AlertTitle, Box, Grid, Stack } from '@mui/material';
-import { isEmpty } from 'lodash-es';
+import { Box, Grid, Stack } from '@mui/material';
 
+import ClientAlerts from '@/modules/client/components/ClientAlerts';
 import ClientCustomDataElementsCard from '@/modules/client/components/ClientCustomDataElementsCard';
 import ClientEnrollmentCard from '@/modules/client/components/ClientEnrollmentCard';
 import ClientProfileCard from '@/modules/client/components/ClientProfileCard';
 import { ClientFieldsFragment } from '@/types/gqlTypes';
 
 export interface Props {
   client: ClientFieldsFragment;
-  notices?: ({
-    id: string;
-    content: React.ReactNode;
-    title?: React.ReactNode;
-  } & Omit<AlertProps, 'children'>)[];
 }
 
-const ProfileLayout: React.FC<Props> = ({ client, notices = [] }) => {
+const ProfileLayout: React.FC<Props> = ({ client }) => {
   const canViewEnrollments = client.access.canViewEnrollmentDetails;
+  const canViewClientAlerts = client.access.canViewClientAlerts;
   return (
     <Box data-testid='clientProfile'>
       <Grid container spacing={2}>
-        {!isEmpty(notices) && (
-          <Grid item xs={12}>
-            <Grid container spacing={2} justifyContent='space-between'>
-              {notices.map(({ content, title, id, ...props }) => (
-                <Grid item xs={12}>
-                  <Alert variant='filled' {...props} key={id}>
-                    {title && <AlertTitle>{title}</AlertTitle>}
-                    {content}
-                  </Alert>
-                </Grid>
-              ))}
-            </Grid>
-          </Grid>
-        )}
         <Grid item md={12} lg={canViewEnrollments ? 6 : 8}>
           <Stack gap={2}>
             <ClientProfileCard client={client} />
             <ClientCustomDataElementsCard client={client} />
           </Stack>
         </Grid>
-        {canViewEnrollments && (
-          <Grid item md={12} lg={6}>
-            <ClientEnrollmentCard client={client} />
-          </Grid>
-        )}
+        <Grid item md={12} lg={6}>
+          <Stack gap={2}>
+            {canViewClientAlerts && <ClientAlerts client={client} />}
```

Comment:
I saw in the Figma file "This should not be shown if the alert feature is not enabled for the installation" -- are there other examples of installation-wide feature switches in our codebase that I can check out to understand the best way to do this?

Reply:
Since we have a special permission for viewing client alerts, we'll just use that permission as a feature toggle. That's usually what we do. So it should really read, "This should not be shown if the user does not have permission to view client alerts for this client"

Reply:
Thanks, that makes sense!